pkg/client/client.go

```diff
@@ -618,24 +618,41 @@ func NewClient(ctrlRuntimeClient client.Client, secretName, namespace, regionID
 
 //Init alibabacloud configuration
 //https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/auth/credentials/providers/configuration.go
-func newConfiguration(ctrlRuntimeClient client.Client, secretName, namespace string, configManagedClient client.Client) (*providers.Configuration, error) {
-	config := &providers.Configuration{}
-
-	if secretName != "" {
-		var secret corev1.Secret
-		if err := ctrlRuntimeClient.Get(context.Background(), client.ObjectKey{Namespace: namespace, Name: secretName}, &secret); err != nil {
-			if apimachineryerrors.IsNotFound(err) {
-				return nil, machineapiapierrors.InvalidMachineConfiguration("alibabacloud credentials secret %s/%s: %v not found", namespace, secretName, err)
-			}
-			return nil, err
-		}
-		err := fetchCredentialsFileFromSecret(&secret, config)
```

Comment:
looks like this function is now unused, and some of the constants at the top of this file as well.

Reply:
Removed and cleaned up.